infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -106,12 +119,20 @@
      */
     public StreamLogFiles(ServerContext serverContext, boolean noVerify) {
         logDir = Paths.get(serverContext.getServerConfig().get("--log-path").toString(), "log");
+        initStreamLogDirectory();
+
+        String logSizeQuotaParam = serverContext.getServerConfig().get("--log-size-quota").toString();
+        logSizeLimit = Long.parseLong(logSizeQuotaParam);
+        long initialLogSize = estimateSize(logDir);
+        logSizeQuota = new AtomicReference<>(new ResourceQuota("LogSizeQuota",
+                initialLogSize, logSizeLimit));
```

Comment:
On startup, if we come up in the state of exceeded quota, should we unless log that? cause maybe clients are already backing off, to understand why writes are not coming into this server.